src/commands/get-organization-by-id.ts

```diff
@@ -0,0 +1,64 @@
+import {BaseCommand} from '../lib'
+import {getEnvironment} from '../lib/environments'
+import {flags} from '@oclif/command'
+
+const GET_ORGANIZATION_BY_ID = `
+query GetOrganizationById($orgID: ID!) {
+  getOrganizationByID(orgID: $orgID) {
+    uid
+    name
+    members {
+      uid
+      auth0User {
+        name
+        email
+        id
+      }
+    }
+    createdBy {
+      auth0User {
+        name
+        email
+        id
+      }
+    }
+  }
+}
+`
+
+export default class GetOrganizationByID extends BaseCommand {
+  static description = 'Get an Organization by its UID'
+
+  static examples = [
+    '$ slash-graphql get-organization-by-id 0x123',
```

Comment:
'$ slash-graphql get-organization-by-id -o 0x123',
This won't work without -o as it's a flag, not arg.